content/en/1.13.x/concepts/updating-bottlerocket.markdown

```diff
@@ -0,0 +1,324 @@
+3. Go to the "Compute" tab.
+4. Under "Node groups", look for the node group that you want to update.
+5. In the "AMI Release Version" column, click on "Update now".
+6. A modal box will pop up.
+Choose your desired update strategy (e.g. "Rolling update") and press the "Update" button.
+
+This will update your Bottlerocket nodegroup, and subsequently your Bottlerocket nodes, to the latest version of Bottlerocket.
+
+Detailed instructions can be found in the [EKS User Guide, in the "AWS Management Console" tab](https://docs.aws.amazon.com/eks/latest/userguide/update-managed-node-group.html#mng-update).
+
+##### `eksctl`
+
+As described in the [EKS User Guide](https://docs.aws.amazon.com/eks/latest/userguide/update-managed-node-group.html#mng-update), you can also use `eksctl` to update your Bottlerocket nodes.
+For example, you can run the following command to update your Bottlerocket nodes to the latest version of Bottlerocket, replacing `NODEGROUP_NAME`, `CLUSTER_NAME`, and `REGION_CODE` with your own values:
+
+```bash
+eksctl upgrade nodegroup \
+  --name=NODEGROUP_NAME \
+  --cluster=CLUSTER_NAME \
+  --region=REGION_CODE
+```
+
+Further details and notes are available in the EKS User Guide linked above.
+
+### SSM
+
+If your Bottlerocket nodes are registered with AWS Systems Manager (SSM), you can use SSM Command Documents to update your Bottlerocket nodes.
+
+**Important:** when using the SSM instructions below to update Bottlerocket nodes, workloads will NOT be drained from the node before rebooting the node (unlike when using Brupop, for example).
+
+#### SSM Command Document: Update Bottlerocket
+
+SSM Command Documents allow you to specify shell commands to run on target nodes.
+In our case, we will use the `aws:runShellScript` SSM Action to run the `apiclient update` command on our Bottlerocket nodes.
+Please see the [`apiclient` documentation](https://github.com/bottlerocket-os/bottlerocket/blob/develop/sources/api/apiclient/README.md#update-mode) to learn more about `apiclient update`.
+
+The following two subsections will cover how to create two SSM Command Documents: one Command Document for preparing a Bottlerocket node to use the latest available version of the Bottlerocket OS image, and one Command Document for rebooting a Bottlerocket node (in order to reboot into and use that latest acquired Bottlerocket OS image).
+
+Both steps can be combined into a single SSM Command Document, however keeping the two steps separate allows you to apply updates and reboot your Bottlerocket nodes in separate patterns, if needed.
+For example, you may want to apply updates to your Bottlerocket nodes in groups, but reboot them in a rolling pattern.
+
+##### 1. Create the SSM Command Document for Updating
+
+The following steps create an SSM Command Document that will update a Bottlerocket node to the latest version of Bottlerocket.
+
+1. Go to the [SSM Console](https://console.aws.amazon.com/systems-manager/).
```

Comment:
Keep looking for "how to create SSM Command Documents" docs on the AWS docs site

Reply:
found